### ISSUE TITLE
Adding SkiaSharp.Views.Uno to Wpf project results in crash at runtime.

### ISSUE THREAD

**Issue by MaksimNikonovOrioninc** [unoplatform/uno#5991]
## Current behavior
On attempt to run WPF project that contains reference on SkiaSharp.Views.Uno application crashing at runtime with error 

System.IO.FileNotFoundException: 'Could not load file or assembly 'SkiaSharp.Views.UWP, Version=2.80.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.'

## Expected behavior

Application should run

## How to reproduce it (as minimally and precisely as possible)

Create a new uno solution with WPF project inside.
Add a reference to the SkiaSharp.Views.Uno nuget package to the Wpf project.

## Workaround

No workaround

## Environment

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [x] Other: SkiaSharp.Views.Uno

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [x] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: 16.9.2)
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

Also, Probably this issue is related [#4713](https://github.com/unoplatform/uno/issues/4713) and if I understood everything properly this pool request contains fixes for this issue https://github.com/mono/SkiaSharp/pull/1634

so, question, is there is any plans to release that to public? In project, on whitch we work, actively used  SkiaSharp and we need an ability to release app to some corporates customers as single exe file, because MS Store disabled on their side and there is no way to distribute application to them, the workaround in our case could be creation of WPF build!

**Comment by jeromelaban** [comment]
Thanks for the report. You found it properly, https://github.com/unoplatform/uno/issues/4713 is needed for this to work. At this point, we're waiting on the SkiaSharp team to accept the PR. 

This issue is a duplicate of https://github.com/unoplatform/uno/issues/4713, I'll close this one.
